init.lua

```diff
@@ -292,7 +292,7 @@ travelnet.update_formspec = function( pos, puncher_name, fields )
 
 
       travelnet.reset_formspec( meta );
-      travelnet.show_message( pos, puncher_name, "Error", S("Update failed! Resetting this box on the travelnet."));
+      travelnet.show_message( pos, puncher_name, "Error", S("Update failed! Resetting...."));
```

Comment:
```suggestion
      travelnet.show_message( pos, puncher_name, "Error", S("Update failed, restarting."));
```

Reply:
But it doesn't restart--it resets (reverts the attempted setting change). Also, the string "Update failed! Resetting..." contains a sentence and a popup-like interjection, but having a comma in there creates a run-on sentence (two phrases jammed together).

Reply:
Ugh, both files should have three dots not four. I'll fix that.

Reply:
> But it doesn't restart--it resets (reverts the attempted setting change). Also, the string "Update failed! Resetting..." contains a sentence and a popup-like interjection, but having a comma in there creates a run-on sentence (two phrases jammed together).

My bad, sorry for this. I guess I should be more careful on the future.
It looks good now. 👍